pkg/rescheduler/strategies/lownodeutilization.go

```diff
@@ -90,16 +95,16 @@ func LowNodeUtilization(client clientset.Interface, strategy api.ReschedulerStra
 
 	if len(lowNodes) == 0 {
 		fmt.Printf("No node is underutilized\n")
-		return
+		return 0
 	} else if len(lowNodes) < strategy.Params.NodeResourceUtilizationThresholds.NumberOfNodes {
 		fmt.Printf("number of nodes underutilized is less than NumberOfNodes\n")
-		return
+		return 0
 	} else if len(lowNodes) == len(nodes) {
 		fmt.Printf("all nodes are underutilized\n")
-		return
+		return 0
 	} else if len(targetNodes) == 0 {
 		fmt.Printf("no node is above target utilization\n")
-		return
+		return 0
 	}
 	SortNodesByUsage(targetNodes)
```

Comment:
Also I think move everything above in the original function. And move stuff below in the new function.

Reply:
Well, I actually thought about that but 
- We have the pass just the values like lowNodes,TargetNodes etc to the evictPodsFromOverUtilized function not client etc that we will be building as part of unit tests.
- Just to be consistent with other strategy where we have a registration function and it delegates to other function which does everything else.